src/Env.elm

```diff
@@ -31,12 +33,13 @@ type alias Flags =
     }
 
 
-init : Flags -> Perspective -> Env
-init flags perspective =
+init : Flags -> Nav.Key -> Perspective -> Env
+init flags navKey perspective =
     { operatingSystem = operatingSystemFromString flags.operatingSystem
     , basePath = flags.basePath
     , apiBasePath = ApiBasePath flags.apiBasePath
     , appContext = AppContext.fromString flags.appContext
+    , navKey = navKey
```

Comment:
Moved the navKey out of the App model and into Env so that we can navigate from any page.